app/services/reCaptcha.js

```diff
@@ -0,0 +1,87 @@
+let reCaptchaPromise = undefined;
+let reCaptchaIntiPromise = undefined;
+let gAssignedId = undefined;
+
+export function initializeRecaptcha(elementId, sitekey){
+
+    if(gAssignedId){
+        console.warn(`Recaptcha already initialized, ${gAssignedId}`)
+        return;
```

Comment:
to be consistent return a resolved promise with i
```suggestion
        return Promise.resolve(gAssignedId);
```